lib/OpenQA/Schema/ResultSet/Assets.pm

```diff
@@ -40,15 +40,19 @@ sub register {
     my ($self, $type, $name, $missingok) = @_;
     $missingok //= 0;
 
+    unless ($name) {
+        log_warning "attempt to register asset with empty name";
```

Comment:
Hm, we had "log_info" in other cases and probably same reasoning as above applies: If it's an user error we should propagate to the user, otherwise info or debug, not warn, right?

Reply:
This is just a sanity check which actually could be removed because the PR actually ensures that this method is not called with an empty name. If that would be true, the user hasn't done anything wrong.

Reply:
It seems very strange that a sanity check would log a warning instreaf of throwing an exception.